master/static/migrations/20220423162505_generic-checkpoints.tx.up.sql

```diff
@@ -19,7 +19,7 @@ CREATE OR REPLACE VIEW public.checkpoints_view AS
         c.id AS id,
         c.uuid AS uuid,
         t.task_id,
-        NULL as allocation_id, -- TODO, is there any trickery to deduce allocation_id?
+        t.task_id||'.'||t.run_id as allocation_id,
```

Comment:
I don't think this is right. this would associated every checkpoint with the last run

Reply:
I think it is truly null for old enough experiments

Reply:
is `raw_checkpoints.trial_run_id` the right thing to use then?

Reply:
Reflecting our Slack conversation:
`raw_checkpoints.trial_run_id` _is_ there right thing to use here. However, older checkpoints may still have trouble loading because tasks and allocations did not exist until a certain release. So task_id and allocation_id would conceivably still be null. We decided I would look into the repercussions of making those nullable fields.